angular.json

```diff
@@ -34,42 +50,36 @@
                 "output": "./static/media"
               }
             ],
-            "styles": [
-              "src/styles.css"
-              ]
-            },
-          
+            "styles": ["src/styles.css"]
+          },
           "configurations": {
             "production": {
-              "indexTransform": "./index-html-transform.js",
-              "sourceMap": false,
-              "customWebpackConfig": {
-                  "path": "./extra-webpack.config.js",
-                  "mergeRules": {
-                    "output": "replace",
-                    "entry": "replace",
-                    "plugins": "append"
-                  }
+              "plugins": ["./extra-esbuild.config.js"],
+              "indexHtmlTransformer": "./index-html-transform.js",
+              "fileReplacements": [
+                {
+                  "replace": "src/environments/environment.ts",
+                  "with": "src/environments/environment.prod.ts"
                 }
+              ]
             },
             "development": {
-              "buildOptimizer": false,
               "optimization": false,
               "sourceMap": true,
               "namedChunks": true,
               "styles": [
-                "src/keycloak-theme/assets/keycloak-resources/login/resources/resources-common/node_modules/@patternfly/patternfly/patternfly.min.css",
-                "src/keycloak-theme/assets/keycloak-resources/login/resources/resources-common/node_modules/patternfly/dist/css/patternfly.min.css",
-                "src/keycloak-theme/assets/keycloak-resources/login/resources/resources-common/node_modules/patternfly/dist/css/patternfly-additions.min.css",
-                "src/keycloak-theme/assets/keycloak-resources/login/resources/css/login.css",
-                "src/keycloak-theme/assets/keycloak-resources/login/resources/resources-common/lib/pficon/pficon.css"
+                "src/keycloak-theme/assets/.keycloakify/login/resources-common/node_modules/@patternfly/patternfly/patternfly.min.css",
+                "src/keycloak-theme/assets/.keycloakify/login/resources-common/node_modules/patternfly/dist/css/patternfly.min.css",
+                "src/keycloak-theme/assets/.keycloakify/login/resources-common/node_modules/patternfly/dist/css/patternfly-additions.min.css",
+                "src/keycloak-theme/assets/.keycloakify/login/css/login.css",
+                "src/keycloak-theme/assets/.keycloakify/login/resources-common/lib/pficon/pficon.css"
```

Comment:
Nice catch.  

I might change this again though because I realized that github pages does not serves the dotfiles.  
This might be a problem for pepole wanting to publish their storybook.  

I'll submit a PR if I do.